server/src/main/java/org/apache/iotdb/db/metadata/schemainfo/PathsUsingTemplateInfo.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.db.metadata.schemainfo;
+
+public class PathsUsingTemplateInfo implements ISchemaInfo {
+  private final String templateName;
+
+  public PathsUsingTemplateInfo(String templateName) {
+    this.templateName = templateName;
+  }
```

Comment:
The returned result is paths using template. It is path rather than templateName. 

The schema template is used on device node. Maybe we can return a DeviceSchemaInfo and let the operator choose whether the device is one of the target results.